src/main/java/de/zalando/tip/zalenium/proxy/DockerSeleniumRemoteProxy.java

```diff
@@ -137,11 +150,16 @@ public int getAmountOfExecutedTests() {
     }
 
     public void videoRecording(final VideoRecordingAction action) {
-        try {
-            String containerId = getContainerId();
-            processVideoAction(action, containerId);
-        } catch (Exception e) {
-            LOGGER.log(Level.SEVERE, getNodeIpAndPort() + e.toString(), e);
+        if (isVideoRecordingEnabled()) {
+            try {
+                String containerId = getContainerId();
+                processVideoAction(action, containerId);
+            } catch (Exception e) {
+                LOGGER.log(Level.SEVERE, getNodeIpAndPort() + e.toString(), e);
+            }
+        } else {
+            String message = String.format("%s: Video recording is disabled", action.getRecordingAction());
+            LOGGER.log(Level.INFO, "{0} {1}", new Object[]{getNodeIpAndPort(), message});
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "{0} {1}" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)